src/antelope/stores/account.ts

```diff
@@ -256,7 +257,7 @@ export const useAccountStore = defineStore(store_name, {
                             network,
                         });
                     } else {
-                        const authenticator = getAntelope().wallets.getAutenticator(autoLogin)?.newInstance(label);
+                        const authenticator = getAntelope().wallets.getAuthenticator(autoLogin)?.newInstance(label);
```

Comment:
oopss!! :see_no_evil: that was me. Thanks! :+1:

Reply:
Telos cloud wallet logo

![cloud-wallet-icon](https://github.com/telosnetwork/telos-wallet/assets/2939354/3cc2a5bb-6f95-4ce6-b8ab-3236126df0b9)